scripts/common.sh

```diff
@@ -0,0 +1,15 @@
+NODE1_NAME=node1
+NODE1_UUID=16e7d8a7-bfa9-428b-9117-363341bb330b
```

Comment:
No, but if not required I don't want to pin them. Same reason we don't pin a serial id and other unused things.